src/prelude.rs

```diff
@@ -0,0 +1,3 @@
+//! Prelude
+
+pub use embedded_hal::prelude::*;
```

Comment:
Should we rename the traits to something unique to avoid collisions with other `embedded-hal` exports ? Would that work okay ? In that case we would only re-export the delays traits.

Reply:
I checked this and it works even with collisions (double-imported from `stm32l4xx-hal` and `cortex-m`). I'm not sure it works like this on all rust versions >=MSRV, thugh.

Reply:
I checked on 1.31.0 with a fake hal and it works too. However, importing prelude does *not* work on 1.31.0 if `cortex-m` and hal use different versions of `embedded-hal` (I used `1.0.0-alpha.1` in my fake hal to check). Works fine on stable, though.

Reply:
This seems fine them.